dataland-frontend/cypress.config.ts

```diff
@@ -33,7 +33,8 @@ export default defineConfig({
         KEYCLOAK_ADMIN: process.env.KEYCLOAK_ADMIN,
         PGADMIN_PASSWORD: process.env.PGADMIN_PASSWORD,
         RABBITMQ_PASS: process.env.RABBITMQ_PASS,
-        RABBITMQ_USER: process.env.RABBITMQ_USER
+        RABBITMQ_USER: process.env.RABBITMQ_USER,
+        DATA_REQUEST_UPLOAD_MAX_FILE_SIZE_IN_BYTES: process.env.DATA_REQUEST_UPLOAD_MAX_FILE_SIZE_IN_BYTES // TODO this needs to be fixed since this doesnt run in a container!
```

Comment:
Maybe just add ` ?? defaultValue` the end of the line

Reply:
I removed this TODO since it was wrong.  It works now.     Still change needed?

Reply:
Pls resolve as soon as your check if it acutally works was succesful

Reply:
Es scheint immer auf den default value zu springen